setup.R

```diff
@@ -1,6 +1,8 @@
+#!/bin/env Rscript
```

Comment:
Habit when writing *.R scripts. An alternative is something like
```
#!/bin/echo Source me don't execute
```

if a user executes from the shell without specifying explicitly to use `Rscript`

I will remove the shebang.